sandbox/technical_user_guide.md

```diff
@@ -136,17 +131,57 @@ Basic Mzg0MWM1OTQtYThjMC00MWU1LTk4Y2MtMzhiYjQ1MzYwZDNjOmY5NzgwZDMyMzU4OGYxY2RmYz
 * `Accept: application/json`
 * `Authorization: <Encoded Basic authentication>`
 
-**cURL command**
+**cURL commands**
+
+You can choose one of two cURL commands to use.
+
+**Option 1, which performs Basic authentication by passing in an additional header**
+
+Format:
+
+```
+curl -H "authorization: Basic [base64-encoded clientId:secret]"
+```
+
+Example:
 
 ```
 curl -d '' -X POST "https://sandbox.bcda.cms.gov/auth/token" -H "accept: application/json" -H "authorization: Basic Mzg0MWM1OTQtYThjMC00MWU1LTk4Y2MtMzhiYjQ1MzYwZDNjOm\
 Y5NzgwZDMyMzU4OGYxY2RmYzNlNjNlOTVhOGNiZGNkZDQ3NjAy\
 ZmY0OGE1MzdiNTFkYzVkNzgzNGJmNDY2NDE2YTcxNmJkNDUwOGU5MDRh"
 ```
+
+**Option 2, which does only requires you to submit your clientId and secret**
+
+
+Format:
+
+```
+curl --user clientId:secret
+```
+
+Example:
+
+```
+curl --user 3841c594-a8c0-41e5-98cc-38bb45360d3c:f9780d323588f1cdfc3e63e95a8cbdcdd47602ff48a537b51dc5d7834bf466416a716bd4508e904a
```

Comment:
The example should probably include the appropriate headers, body, action, and destination address; similar to the example in option 1.

Reply:
Hmm, I'm actually unsure how the Option 1 example follows the Option 1 format -- to me it looks like the Option 1 example has a lot of extras not included in the Option 1 format (ex: x flags at the beginning, no colon separating the clientId and secret).  Suspect that this is related to the fact that I do not know what you mean by headers, body, action, and destination address.  Could you help me understand?

Reply:
Accepted, resolved.  I tried the cURL command in terminal and did not get an error message.